urbanaccess/gtfs/network.py

```diff
@@ -124,55 +141,89 @@ def create_transit_net(gtfsfeeds_dfs, day,
         gtfsfeeds_dfs.stop_times_int = _time_difference(
             stop_times_df=gtfsfeeds_dfs.stop_times_int)
 
+    # direction_id is conditional, though
+    if 'direction_id' not in gtfsfeeds_df.trips.columns:
+        columns.remove('direction_id')
+    
+    trips_subgdf = gtfsfeeds_df.trips[columns]
+    calendar_selected_trips_df = tripschedualselector(
+                                    input_trips_df=trips_subgdf,
+                                    input_calendar_df=gtfsfeeds_df.calendar,
+                                    day=day)
+
+    # confirm if need to interpolate stop times and, if yes, then do so
+    interpolate_trigger = (gtfsfeeds_df.stop_times_int.empty or 
+                           overwrite_existing_stop_times_int or 
+                           use_existing_stop_times_int == False)
+    if interpolate_trigger:
+        interp_stops = interpolatestoptimes(
+                        stop_times_df=gtfsfeeds_df.stop_times,
+                        calendar_selected_trips_df=calendar_selected_trips_df,
+                        day=day)
+
+        time_diff_interp_stops = timedifference(stop_times_df=interp_stops)
+        gtfsfeeds_df.stop_times_int = time_diff_interp_stops
+
+        # check if we want to save this output to hdf5 file
         if save_processed_gtfs:
-            save_processed_gtfs_data(gtfsfeeds_dfs=gtfsfeeds_dfs,
-                                     dir=save_dir, filename=save_filename)
+            save_processed_gtfs_data(
+                                gtfsfeeds_df=gtfsfeeds_df,
```

Comment:
I think a search and replace may have caused a mistake in your variable naming. I think everywhere you have `gtfsfeeds_df` it should be `gtfsfeeds_dfs`.